Updated from Windows App SDK 1.6 to 1.8

licon4812 opened the pull request:
resolves #284

licon4812 committed:
WindowsAppSDK 1.6 to WindowsAppSDK .WinUI 1.8

licon4812 committed:
Update WinUI.TableView.csproj

licon4812 committed:
Removed full WindowsAppSDK use WinUI metapackage

licon4812 committed:
Merge remote-tracking branch 'upstream/main' into win-app-sdk-1.8

w-ahmad commented:
Thanks for the PR @licon4812! It seems your branch is quite behind the main, could you update it please?

<img width="924" height="68" alt="image" src="https://github.com/user-attachments/assets/3e5d818f-dc2d-4c2e-9dd5-b2aecfbc196a" />

w-ahmad commented on src/WinUI.TableView.csproj:
```diff
@@ -30,8 +30,8 @@
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.250108002" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2903.40" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.WindowsAppSDK.WinUI" Version="1.8.251222000" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.3179.45" PrivateAssets="all" />
```
Do we still need the WebView2 package? I added this package explicitly to fix #170 in PR #182, but I don’t think we’ll need it anymore after migrating to v1.8.

licon4812 replied:
> Do we still need the WebView2 package? I added this package explicitly to fix #170 in PR #182, but I don’t think we’ll need it anymore after migrating to v1.8.

I have now removed references to webview 2. I only left it in because I thought it was needed

licon4812 committed:
Merge remote-tracking branch 'upstream/main' into win-app-sdk-1.8

licon4812 committed:
Removed webview 2 package for windows tfm

w-ahmad commented:
@licon4812  I see your merge commit, but not sure why... Your branch still seems behind. Make sure it's upto date so then we can investigate at what point the actions are failing. 

<img width="1080" height="553" alt="Screenshot_20260415_201848_Edge" src="https://github.com/user-attachments/assets/7884dec9-6ac2-4075-af58-22deefa80f7c" />

licon4812 committed:
Merge branch 'w-ahmad:main' into win-app-sdk-1.8

licon4812 commented:
> @licon4812 I see your merge commit, but not sure why... Your branch still seems behind. Make sure it's upto date so then we can investigate at what point the actions are failing.
> 
> <img alt="Screenshot_20260415_201848_Edge" width="1080" height="553" src="https://private-user-images.githubusercontent.com/17172092/578706421-7884dec9-6ac2-4075-af58-22deefa80f7c.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzYyODg4MDYsIm5iZiI6MTc3NjI4ODUwNiwicGF0aCI6Ii8xNzE3MjA5Mi81Nzg3MDY0MjEtNzg4NGRlYzktNmFjMi00MDc1LWFmNTgtMjJkZWVmYTgwZjdjLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDE1VDIxMjgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0NDIyZmJlNzJkNDg2OTQ2MmE5MzEyZGQ5MzQ5ZTg5MzE5ODgwYzA4MDRiY2EwZmVjZWY3OTRmNmIxNWE5NWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRmpwZWcifQ.xUc1KXrgghmQY0jURUZlo0xKj-GVFxYNASA_HnfXSEg">

It is bizarre that I had to do it twice 🤣

w-ahmad reviewed (changes requested):
The tests are failing because the project doesn’t include the WinAppSDK package and the sample apps are failing because they are on v1.6 too. Since the TableView project has moved to 1.8 and now only use the WinUI package, we need to add the WinAppSDK and Build Tools packages to the test and samples project.

licon4812 committed:
Added WindowsAppSDK to test and sample projects

licon4812 committed:
Added MSIX build tools to test project

licon4812 committed:
Added Microsoft.Windows.SDK.BuildTools to Tests

licon4812 committed:
Update WinUI.TableView.Tests.csproj

licon4812 commented:
@w-ahmad  Currently, the tests work locally, just not on the pipeline agent

<img width="990" height="380" alt="image" src="https://github.com/user-attachments/assets/2c461c7d-5e34-4879-b2a8-0c51863ac1ea" />

licon4812 committed:
Update ci-build.yml

licon4812 committed:
Revert "Update ci-build.yml"

This reverts commit 0435addfe36fbcf0f5692dbed5cca31b2a81572e.

w-ahmad committed:
Update CI build environment to windows-2025-vs2026

licon4812 committed:
Merge branch 'w-ahmad:main' into win-app-sdk-1.8

w-ahmad committed:
Revert "Update CI build environment to windows-2025-vs2026"

This reverts commit fb688be03b7d01ba6d0ad282db53343212956abd.

w-ahmad committed:
Remove PublishProfile and update TestHost package usage

Removed the <PublishProfile> property from the test project file. Updated Microsoft.TestPlatform.TestHost reference to exclude build assets, preventing unnecessary build-time dependencies.

w-ahmad committed:
Update UnitTestApp to run tests on launch via command line

Replaces MainWindow.Activate() with UnitTestClient.Run(Environment.CommandLine) in OnLaunched, enabling automated test execution using command-line arguments and streamlining the unit test startup process.

w-ahmad committed:
Correct output path and test execution command

Fix output path for build artifacts and update test command.
